explorer/src/components/instruction/system/types.ts

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable @typescript-eslint/no-redeclare */
+
+import { enums, number, pick, string, StructType } from "superstruct";
+import { Pubkey } from "validators/pubkey";
+
+const CreateAccount = pick({
+  source: Pubkey,
+  newAccount: Pubkey,
+  lamports: number(),
```

Comment:
This makes me realize we should probably be returning all the various lamports values as strings (rust type `StringAmount`). This doesn't need to wait on that, but let's fix that up in the future.